src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -263,6 +264,11 @@
 					.examples("on tool break:")
 					.since("2.1.1");
 		}
+		Skript.registerEvent("Item Damage", SimpleEvent.class, PlayerItemDamageEvent.class, "[player] (tool|item) damag(e|ing)", "[player] damag(e|ing) [(a|the)] (tool|item)")
```

Comment:
`on player damaging the tool` this seems a rather redundant syntax, just the first one is fine. Also, `player` is redundant as well since users expect players to trigger this event.

Reply:
Duly noted, will change it up.